pages/base_test.py

```diff
@@ -88,19 +87,15 @@ def delete_run(self, mozwebqa, run, delete_version=False):
         if delete_version:
             self.delete_version(mozwebqa, version=run['version'])
 
-    def create_suite(self, mozwebqa, product, use_API, status='active', case_list=[], **kwargs):
-        if use_API:
-            credentials = mozwebqa.credentials['default']
+    def create_suite(self, mozwebqa, product, api=None, status='active', case_list=[], **kwargs):
+        if api is not None:
```

Comment:
`api` should be an instance of `MoztrapAPI`, so I'm not sure I get the 'empty string' part of your comment. We could add type checking, and `if api` would certainly work, but here I'm checking if anything other than the default has been passed, which shows an intention to use the API. We'd fail badly if this was somehow a string/boolean/etc.